ext/EnzymeExt.jl

```diff
@@ -0,0 +1,12 @@
+module EnzymeExt
+
+using MeshIntegrals
+using Meshes: Geometry
+using Enzyme
+
+function MeshIntegrals.jacobian(
+        geometry::Geometry, ts::V, ::AutoEnzyme) where {V <: Union{AbstractVector, Tuple}}
+    to.(Enzyme.jacobian(Enzyme.Forward, geometry, ts...))
```

Comment:
Would it make sense to include `Enzyme.Forward` as an option to `AutoEnzyme`, such that one could also use `Enzyme.Backward` mode?

Reply:
This is a good point. If one did a reverse mode calc on a higher-level function such as `integral`, would this be an issue or would Enzyme just take care of it? I need to think this through. Ideas?